docs/quickstart-cli.md

```diff
@@ -69,12 +71,22 @@ Make sure to modify the registration.yaml above to set the proper install device
 The SD-card on a Raspberry Pi is usually `/dev/mmcblk0`.
 :::
 
+<Tabs>
+<TabItem value="seedImagex86" label="Seed Image (x86_64)" default>
+<CodeBlock language="yaml" title="seedimage.yaml" showLineNumbers>{SeedImage}</CodeBlock>
+</TabItem>
+<TabItem value="seedImagerpi" label="Seed Image for Raspberry Pi" default>
+The seed image is not yet used for Raspberry Pi and will have to be generated manually in the section [Preparing the seed image manually](./quickstart.md#preparing-the-installation-seed-image)
```

Comment:
This URL seams to be broken within the preview

Reply:
Good catch!